libopenage/renderer/resources/texture_data.h

```diff
@@ -68,4 +73,6 @@ class Texture2dData {
 	std::vector<uint8_t> data;
 };
 
-}}} // namespace openage::renderer::resources
+} // namespace resources
```

Comment:
I think in C++ 17 you can nest multiple namespaces in one statement, like
```
namespace openage::renderer::resources{
}    // namespace openage::renderer::resources
```

Reply:
that's what the change is? :D

Reply:
fixed